VERSION

```diff
@@ -1 +1 @@
-0.4.0
+0.4.4
```

Comment:
Not sure this PR needs to control the version -- that should probably be handled by the maintainers.